doc/changes/changes_1.1.4.md

```diff
@@ -0,0 +1,12 @@
+# Virtual Schema for Document Data in Files on Azure Blob Storage 1.1.4, released 2023-??-??
+
+Code name: Refactored LocalAbsTestSetup
```

Comment:
```suggestion
Code name: Refactoring of integration tests
```
The user is not interested in internal refactorings. If there are no fixed vulnerabilities we don't need a release I guess